lib/flv/coalesce-stream.js

```diff
@@ -35,7 +35,7 @@ var CoalesceStream = function(options) {
   this.push = function(output) {
     // buffer incoming captions until the associated video segment
     // finishes
-    if (output.text) {
+    if (output.content || output.text) {
```

Comment:
I believe other types of captions (708) could still use text as of now, so both content and text are included.